legacy/PimRecordListHelper.js

```diff
@@ -717,27 +721,45 @@ async function addExportColumns(
           field = field.slice(11, -1);
           exportColumns.push({
             fieldName: defaultColumns.get(field),
-            label: templateHeaders[i],
+            label: templateHeaders[lastHeaderRowIndex][i],
```

Comment:
do we need to make sure `templateHeaders[lastHeaderRowIndex]` is never null

Reply:
Because `lastHeaderRowIndex = templateHeaders.length - 1` we wont get array index out of bounds